packages/jasmine/index.bzl

```diff
@@ -0,0 +1,6 @@
+# Public API surface re-exports
```

Comment:
@bazel/karma and @bazel/typescript use defs.bzl. Should be consistent. I prefer index.bzl myself since its more node-ish. I think we should change to index.bzl for @bazel/karma & @bazel/typescript. Now's a good time since we've just renamed the workspaces.

Reply:
We could have both defs.bzl and index.bzl for those for a few releases before removing the defs..

Reply:
sounds good